eng/intellisense.targets

```diff
@@ -10,6 +10,11 @@
     <IntellisensePackageXmlFile Condition="'$(IntellisensePackageXmlFile)' == '' and '$(UseIntellisensePackageDocXmlFile)' == 'true' and Exists($(IntellisensePackageXmlFileFromDotNetPlatExtFolder))">$(IntellisensePackageXmlFileFromDotNetPlatExtFolder)</IntellisensePackageXmlFile>
   </PropertyGroup>
 
+  <Target Name="VerifyAssemblySupportsDocsXmlGeneration">
+    <Error Condition="'$(UseIntellisensePackageDocXmlFile)' == 'false' and '$(IsPartialFacadeAssembly)' == 'true'" Text="Cannot currently generate full xml documentation for an assembly that is a partial facade." />
+    <Error Condition="'$(UseIntellisensePackageDocXmlFile)' == 'false' and '$(GeneratePlatformNotSupportedAssemblyMessage)' != ''" Text="Cannot currently generate full xml documentation for an assembly that generates PNSE." />
```

Comment:
Instead of saying `generates PNSE`, we should say `throws PlatformNotSupportedException`.